build.sbt

```diff
@@ -66,4 +67,5 @@ lazy val examples = project
     libraryDependencies ++= sparkDependencies
   )
   .settings(parallelExecution in Test := false)
+  .settings(fork in Test := true)
```

Comment:
I added these to the 'spark' and 'examples' modules in the hope that this would reduce the number of times I have to restart `sbt` due to out of memory errors while developing.

Reply:
why not add it to and use the `testSettings` val ?

Reply:
I don't want it on the non-Spark projects as it's slower and those projects don't have the issue.